crates/red_knot_python_semantic/resources/mdtest/sys_version_info_310.md

```diff
@@ -0,0 +1,13 @@
+# `sys.version_info` for Python 3.13
+
+This test makes sure that we correctly parse the `target-version` configuration option. See
+`sys_version_info.md` for the actual tests for `sys.version_info`.
```

Comment:
nit: It might be good to be a bit more explicit that this is also a unit test for the `red_knot_test` crate in some ways.

Reply:
This is exactly what this comment was all about, but I can try to make it more explicit :smile:

Reply:
Yeah I think all I'm really asking for is

```suggestion
This test makes sure that `red_knot_test` correctly parses the `target-version` option in a
toml configuration block. See `sys_version_info.md` for the actual tests for
`sys.version_info`.
```